### PR TITLE
Remove the sso submodule

### PR DESCRIPTION
**Pull request by heiko-braun**

### CODE REVIEW

**Comment by heiko-braun** [comment]
@Ladicek mind a review?

File: pom-redhat.xml [hunk 1]
```diff
@@ -48,34 +48,300 @@
...
         <version>${version.wildfly.swarm}</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+
       <dependency>
         <groupId>org.keycloak</groupId>
         <artifactId>keycloak-authz-client</artifactId>
-        <version>2.5.7.Final-redhat-2</version>
+        <version>2.5.5.Final</version>
```
Comment (Ladicek) [inline pom-redhat.xml:63]: This is `pom-redhat.xml`, we probably want to use the productized Keycloak stuff?

File: pom-redhat.xml [hunk 2]
```diff
@@ -14,31 +14,31 @@
...
     <version>10</version>
   </parent>
 
-  <groupId>io.openshift.booster</groupId>
-  <artifactId>wfswarm-rest-http-secured-parent</artifactId>
-  <version>5-SNAPSHOT</version>
-  <packaging>pom</packaging>
+  <packaging>war</packaging>
 
   <name>Secured HTTP Example - Parent</name>
```
Comment (Ladicek) [inline pom-redhat.xml]: Probably wanna change.

File: pom-redhat.xml [hunk 3]
```diff
@@ -48,34 +48,300 @@
...
   </dependencyManagement>
 
-   <build>
-      <plugins>
-         <plugin>
-           <groupId>org.apache.maven.plugins</groupId>
-           <artifactId>maven-surefire-plugin</artifactId>
-           <configuration>
-             <systemPropertyVariables>
-               <swarm.build.repos>https://maven.repository.redhat.com/ga/,https://maven.repository.redhat.com/earlyaccess/all/</swarm.build.repos>
```
Comment (Ladicek) [inline pom-redhat.xml:74]: This shouldn't be deleted IMHO.

File: pom-redhat.xml [hunk 4]
```diff
@@ -48,34 +48,300 @@
...
+      <artifactId>keycloak-authz-client</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+  </dependencies>
+
+  <build>
+    <resources>
+      <resource>
+        <directory>../src/main/fabric8</directory>
```
Comment (Ladicek) [inline pom-redhat.xml]: This path is wrong.

Also do we need to keep the `<resources>` section at all?

File: pom.xml [hunk 5]
```diff
@@ -14,31 +14,31 @@
...
   ~  limitations under the License.
   -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>wfswarm-rest-http-secured</artifactId>
+  <groupId>io.openshift.booster</groupId>
+  <version>7.0.0-redhat-1-SNAPSHOT</version>
```
Comment (Ladicek) [inline pom.xml]: Did you maybe confuse `pom.xml` and `pom-redhat.xml`? This belongs to `pom-redhat.xml`.

**Comment by Ladicek** [comment]
@heiko-braun Please take a look at the comments.

File: pom-redhat.xml (hunk 1 above)
Comment (heiko-braun) [reply pom-redhat.xml:63]: No, this has been discussed with Ken and we step back from the productised bits here to make it more "certified"

File: pom-redhat.xml (hunk 2 above)
Comment (heiko-braun) [reply pom-redhat.xml]: +1

File: pom-redhat.xml (hunk 3 above)
Comment (heiko-braun) [reply pom-redhat.xml:74]: right, I think @mkouba added this recently

File: pom.xml (hunk 5 above)
Comment (heiko-braun) [reply pom.xml]: good catch

File: pom-redhat.xml (hunk 4 above)
Comment (heiko-braun) [reply pom-redhat.xml]: +1

Comment (heiko-braun) [reply pom-redhat.xml]: but we need to keep it

**Comment by heiko-braun** [comment]
@Ladicek the last commit should address your points

File: pom-redhat.xml (hunk 1 above)
Comment (Ladicek) [reply pom-redhat.xml:63]: Oh, this is the "tech preview" stuff, right?

File: pom.xml [hunk 6]
```diff
@@ -55,8 +55,285 @@
...
+      <artifactId>keycloak-authz-client</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+  </dependencies>
+
+  <build>
+    <resources>
+      <resource>
+        <directory>../src/main/fabric8</directory>
```
Comment (Ladicek) [inline pom.xml]: Wrong path.

**Review by Ladicek — CHANGES_REQUESTED** [review]
One more, otherwise LGTM.

**Comment by heiko-braun** [comment]
@Ladicek done

**Review by Ladicek — APPROVED** [review]

**Comment by Ladicek** [comment]
I will try running it locally, but LGTM for now.

**Comment by Ladicek** [comment]
Verified locally. Thanks @heiko-braun! I really like how the booster implementation looks now.

**Comment by heiko-braun** [comment]
yes, agreed. It's better. thanks for the review

**Pull request merged**